packages/react/src/components/Tile/Tile.tsx

```diff
@@ -166,12 +227,78 @@ ClickableTile.propTypes = {
   rel: PropTypes.string,
 };
 
-export const SelectableTile = React.forwardRef(function SelectableTile(
+export interface SelectableTileProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * The child nodes.
+   */
+  children?: ReactNode;
+
+  /**
+   * The CSS class names.
+   */
+  className?: string;
+
+  /**
+   * Specify whether the SelectableTile should be disabled
+   */
+  disabled?: boolean;
+
+  /**
+   * The ID of the `<input>`.
+   */
+  id?: string;
+
+  /**
+   * The `name` of the `<input>`.
+   */
+  name?: string;
+
+  /**
+   * The empty handler of the `<input>`.
+   */
+  onChange?(event: ChangeEvent<HTMLDivElement>): void;
+
+  /**
+   * Specify the function to run when the SelectableTile is clicked
+   */
+  onClick?(event: MouseEvent<HTMLDivElement>): void;
+
+  /**
+   * Specify the function to run when the SelectableTile is interacted with via a keyboard
+   */
+  onKeyDown?(event: KeyboardEvent<HTMLDivElement>): void;
+
+  /**
+   * `true` to select this tile.
+   */
+  selected?: boolean;
+
+  /**
+   * Specify the tab index of the wrapper element
+   */
+  tabIndex?: number;
+
+  /**
+   * The `title` of the `<input>`.
+   */
+  title?: string;
+
+  /**
+   * The value of the `<input>`.
+   */
+  value: string | number;
+}
+
+export const SelectableTile = React.forwardRef<
+  HTMLDivElement,
+  SelectableTileProps
+>(function SelectableTile(
   {
     children,
     className,
     disabled,
     id,
+    // @ts-expect-error: Deprecated prop
     light,
     name,
```

Comment:
confirmed we're good to remove the `name`, `value` & `disabled` attributes from the div. looks like that was the only thing we were using the `name` and `value` prop for though so I think we can either remove them from the component's destructuring or add an underline (`name: _name`) to the name to avoid these errors 
<img width="707" alt="image" src="https://user-images.githubusercontent.com/40550942/234908822-c40f3495-130c-4c12-88ac-4800a474b699.png">
I'm not sure if removing it from the props since that might be considered a breaking change even if it's not used, maybe we @deprecate them instead? @tay1orjones

Reply:
Yeah let's just deprecate them

Reply:
`disabled` still seems to very much be in use for `className`s though, so I presume that only the other two should really be deprecated… which I'm doing